redmine/redmine_dumper.py

```diff
@@ -97,17 +97,17 @@ def get_events(journal_dict, issue_id):
 
 class RedmineDumper:
 
-    def __init__(self):
+    def __init__(self, database: str, logging_level: int):
         self.logger = logging.getLogger(self.__module__)
-        self.logger.setLevel(logging.DEBUG)
+        self.logger.setLevel(logging_level)
```

Comment:
wonder if the https://github.com/ilausuch/project_management_statistics/pull/22/files#diff-d63843d0e3359f796f659ab22cbf53574f27e5df016166a8e3edba7f7ceeed98R104 left untouched here intentionally? or is  it needed to take `logging_level` as well?

Reply:
done